LICENSE

```diff
@@ -347,9 +347,9 @@ The licences which components of this software fall under are as follows.
 
 3)
   From 1991 to August 1999, ImageMagick was developed and distributed by
-  E. I. du Pont de Nemours and Company:
+  E. I. du Point de Nemours and Company:
 
-  Copyright 1999 E. I. du Pont de Nemours and Company
+  Copyright 1999 E. I. du Point de Nemours and Company
```

Comment:
The company name is "du Pont" not "du Point" and so it is incorrect to "fix" this "error."

Reply:
Indeed, fixed now.